runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayManager.java

```diff
@@ -217,6 +217,17 @@ public synchronized boolean removeIf(Predicate<Overlay> filter)
 		return removeIf;
 	}
 
+	/**
+	 * Returns whether an overlay exists which matches the given predicate.
+	 *
+	 * @param filter Filter predicate function
+	 * @return {@code true} if any overlays match the given filter, {@code false} otherwise
+	 */
+	public synchronized boolean hasOverlay(Predicate<Overlay> filter)
```

Comment:
This should be named .find or anyMatch (rest of methods in this class already follow the naming in java collections)